tests/console/slp.pm

```diff
@@ -22,23 +22,33 @@ use strict;
 use warnings;
 use utils qw(zypper_call systemctl script_retry);
 use Utils::Systemd 'disable_and_stop_service';
+use version_utils 'is_jeos';
 
 sub run {
     my ($self) = @_;
     select_console 'root-console';
 
-    zypper_call 'in openslp openslp-server';
+    # Let's install slpd
+    zypper_call 'in openslp-server';
 
     disable_and_stop_service($self->firewall) if (script_run("which " . $self->firewall) == 0);
 
-    assert_script_run 'useradd -d /var/lib/empty/ -s /sbin/nologin openslp || true';
+    # I guess it is a bug
```

Comment:
This shouldn't be necessary (anymore?) as openslp-server does that itself in its `%post`:

https://build.opensuse.org/package/view_file/SUSE:SLE-15-SP2:GA/openslp/openslp.spec?expand=1

Reply:
Product bug. It's missing `Requires(pre): group(daemon)`.

This should never have been worked around in openQA without reporting it properly :-/